src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const spiderX = spider.clientWidth;
+const spiderY = spider.clientHeight;
+const wallX = wall.clientWidth;
+const wallY = wall.clientHeight;
+const centreX = wallX / 2 - spiderX / 2;
+const centreY = wallY / 2 - spiderY / 2;
+
+spider.style.top = centreX + 'px';
```

Comment:
The assignment of `spider.style.top` should use `centreY` instead of `centreX`. This will correctly position the spider vertically in the center of the wall.